calico_containers/pycalico/datastore.py

```diff
@@ -129,7 +137,15 @@ def ensure_global_config(self):
 
         # Configure IPAM directory structures (to ensure confd is able to
         # watch appropriate directory trees).
-        self.etcd_client.write(CONFIG_IF_PREF_PATH, IF_PREFIX)
+        host = socket.gethostname()
```

Comment:
I don't think we do, because of the line "write_global_config" above that writes the exact same path and value; unless I'm misunderstanding (not without precedent!) this line is effectively a duplicate.